README.md

```diff
@@ -70,40 +68,56 @@ Several individuals across different research projects contribute to the develop
 
-﻿<a href="https://reiner-lemoine-institut.de/"><img src="https://reiner-lemoine-institut.de//wp-content/uploads/2015/09/rlilogo.png" alt="Logo RLI" width="140px"></a> 
+﻿<a href="https://www.iee.fraunhofer.de/"><img src="https://openenergy-platform.org/static/logos/partner/IEE.png" alt="Logo Fraunhofer IEE" width="230px"></a> 
 
-﻿<a href="http://www.znes-flensburg.de/"><img src="https://i2.wp.com/reiner-lemoine-institut.de/wp-content/uploads/2017/03/Logo_ZNES_farbig_NEU.png" alt="Logo ZNES" width="280px"></a> 
+﻿<a href="https://www.oeko.de/"><img src="https://openenergy-platform.org/static/logos/partner/OEI.png" alt="Logo Öko-Institut" width="230px"></a> 
 
-﻿<a href="https://www.uni-flensburg.de/en/"><img src="https://www.uni-flensburg.de/fileadmin/content/system/logos/europa/uni-flensburg-en.png" alt="Logo EUF" width="280px"></a> 
+﻿<a href="https://www.fz-juelich.de"><img src="https://openenergy-platform.org/static/logos/partner/FZJuelich-1.jpg" alt="Logo FZJ" width="230px"></a> 
 
-﻿<a href="http://www.dlr.de/ve/en/desktopdefault.aspx/tabid-12472/21440_read-49440/"><img src="https://openegoproject.files.wordpress.com/2017/02/dlr_logo_vernetzte_energiesysteme_gb_grau.jpg" alt="Logo DLR" width="230px"></a> 
+﻿<a href="https://infai.org/"><img src="https://openenergy-platform.org/static/logos/partner/InfAI-Logo.png" alt="Logo InfAI" width="230px"></a> 
 
-﻿<a href="http://www.hzg.de/index.php.de"><img src="https://i1.wp.com/reiner-lemoine-institut.de/wp-content/uploads/2016/08/logo_hzg_cms10.gif" alt="Logo HZG" width="230px"></a> 
+﻿<a href="https://www.pik-potsdam.de/"><img src="https://openenergy-platform.org/static/logos/partner/pik_logo.png" alt="Logo PIK" width="230px"></a> 
 
-﻿<a href="https://www.iee.fraunhofer.de/"><img src="https://reiner-lemoine-institut.de/wp-content/uploads/2018/02/fraunhofer-iee-1.gif" alt="Logo Fraunhofer IEE" width="230px"></a> 
+﻿<a href="https://www.ier.uni-stuttgart.de/"><img src="https://openenergy-platform.org/static/logos/partner/Logo_IER_UniStuttgart-1200x386.png" alt="Logo IER" width="230px"></a> 
 
-﻿<a href="https://www.oeko.de/"><img src="https://reiner-lemoine-institut.de/wp-content/uploads/2018/02/oeko-institut-logo_transp.png" alt="Logo Öko-Institut" width="230px"></a> 
+﻿<a href="https://www.fortiss.org/"><img src="https://openenergy-platform.org/static/logos/partner/partner_logo_fortiss.png" alt="Logo fortiss" width="230px"></a> 
+
+﻿<a href="https://www.epe.ed.tum.de/ens/startseite/"><img src="https://www.epe.ed.tum.de/typo3/ext/tumtheme/Resources/Public/Icons/Org/tum-logo.png" alt="Logo TUM" width="170px"></a> 
 
 ## Licenses
 
-The Open Energy Family follows the ideal of Open Science. Thus all material (Code, Data, Artwork) will be licensed with suitable open licenses if possible.
+The Open Energy Family follows the ideal of Open Science. Thus all material (Code, Data, Artwork) will be licensed with suitable open licenses.
```

Comment:
Suggest: The Open Energy Family follows the ideal of Open Science. Thus all material (Code, Data, Artwork) is licensed with a suitable open license.